grails-app/services/org/broadinstitute/orsp/QueryService.groovy

```diff
@@ -477,18 +478,23 @@ class QueryService implements Status {
 
     List findIssuesBySearchTermAsProjectKey(String term) {
         // For backwards compatibility with existing "ORSP" prefixes, ignore the prefix and like-clause on the identifier
-        String iLikeTerm = "%" + getIssueNumberFromString(term) + "%"
-        Issue.findAllByProjectKeyIlike(iLikeTerm).collect {
-            [id: it.id,
-             projectKey: it.projectKey,
-             summary: it.summary,
-             reporter: it.reporter,
-             pm: userService.findUsers(it.getPMs()).displayName,
-             actor: userService.findUsers(it.getActorUsernames()).displayName,
-             extraProperties: it.extraPropertiesMap,
-             controller: it.controller,
-             type: it.type
-            ]
+        String issueNumber = getIssueNumberFromString(term)
+        if (!StringUtils.isEmpty(issueNumber)) {
+            String iLikeTerm = "%" + issueNumber + "%"
+            Issue.findAllByProjectKeyIlike(iLikeTerm).collect {
+                [id: it.id,
+                 projectKey: it.projectKey,
+                 summary: it.summary,
+                 reporter: it.reporter,
+                 pm: userService.findUsers(it.getPMs()).displayName,
+                 actor: userService.findUsers(it.getActorUsernames()).displayName,
+                 extraProperties: it.extraPropertiesMap,
+                 controller: it.controller,
+                 type: it.type
+                ]
+            }
+        } else {
+            Collections.emptyList()
```

Comment:
To make this cleaner, you could just fail fast instead of wrapping the whole thing in an if-else. The first line could just be:
```if (StringUtils.isEmpty(issueNumber)) { return Collections.emptyList() }```

Reply:
Also testing out the new multi-line comment feature - pretty cool.